lesson_19/homework_19_1.py

```diff
@@ -0,0 +1,32 @@
+import requests
+
+
+def get_photos(url: str, params: dict):
+    response = requests.get(url, params=params)
+    response.raise_for_status() 
+    data = response.json()
```

Comment:
```suggestion
    data = response.json()
```
а якщо прийде не джсон?